content/docs/tutorials/cni.md

```diff
@@ -0,0 +1,175 @@
++++
+title = "Using CNI"
+weight = 1
```

Comment:
Nit: if we order them by popularity, I would keep the current order and add this to the end.

Reply:
Done